html/includes/graphs/device/processor_separate.inc.php

```diff
@@ -18,7 +18,7 @@
 
 $unit_text = 'Load %';
 
-$units       = '%';
+$units       = '';
```

Comment:
Because it was making them double up; it's already handled in the graph file that's included